docs/pages/technical-specs/expo-updates-1.mdx

```diff
@@ -17,39 +17,45 @@ This is the specification for Expo Updates, a protocol for delivering updates to
 
 Conforming servers and client libraries must fulfill all normative requirements. Conformance requirements are described in this document by both descriptive assertions and key words with clearly defined meanings.
 
-The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in the normative portions of this document are to be interpreted as described in [IETF RFC 2119](https://tools.ietf.org/html/rfc2119). These key words may appear in lowercase and still retain their meaning unless explicitly declared as non‐normative.
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in the normative portions of this document are to be interpreted as described in [IETF RFC 2119](https://tools.ietf.org/html/rfc2119). These key words may appear in lowercase and still retain their meaning unless explicitly declared as non-normative.
 
-A conforming implementation of this protocol may provide additional functionality, but must not where explicitly disallowed or would otherwise result in non‐conformance.
+A conforming implementation of this protocol MAY provide additional functionality, but MUST NOT where explicitly disallowed or would otherwise result in non-conformance.
 
 ### Overview
 
 Conforming servers and client libraries MUST follow the HTTP spec as described in [RFC 7231](https://tools.ietf.org/html/rfc7231) as well as the more precise guidance described in this spec.
 
-An _update_ is defined as a [_manifest_](#manifest-response) together with the assets referenced inside the manifest.
+An _update_ is defined as one or both of the following:
+- a [_manifest_](#manifest-response) together with the assets referenced inside the manifest.
+- a [_directive_](#directive-response).
+
 Expo Updates is a protocol for assembling and delivering updates to apps running on multiple platforms.
 
-An app running a conformant Expo Updates client library MUST load the most recent update saved in the client library's cache, possibly after filtering by the contents of the manifest's [_metadata_](#manifest-response-body).
+An app running a conformant Expo Updates client library MUST load the most recent _manifest_ saved in the client library's cache possibly after filtering by the contents of the manifest's [_metadata_](#manifest-body).
```

Comment:
If a manifest has been saved but the entire update hasn't been fully downloaded, we don't want to load that manifest.

Separately (unrelated to this PR), "cache" is the wrong word here in that a cache implies the data can be arbitrarily erased and repopulated, but in our case we want to make sure that updates are stable.
```suggestion
An app running a conformant Expo Updates client library MUST load the most recent update saved in the client library's update database possibly after filtering by the contents of the manifest's [_metadata_](#manifest-body).
```